### PR TITLE
*: add some privilege columns to mysql.user

### PR DESCRIPTION
**Pull request by tiancaiamao**
Add those columns for issue https://github.com/pingcap/tidb/issues/3323

```
Create_tmp_table_priv
Lock_tables_priv
Create_view_priv
Show_view_priv
Create_routine_priv
Alter_routine_priv
Event_priv
```

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@coocood PTAL

File: bootstrap.go [hunk 1]
```diff
@@ -48,16 +48,23 @@ const (
 		Delete_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
 		Create_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
 		Drop_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
-		Process_priv		ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Process_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
```
Comment (zimulala) [inline bootstrap.go]: Need to align?

File: bootstrap.go [hunk 2]
```diff
@@ -450,6 +462,28 @@ func upgradeToVer12(s Session) {
...
+		"ALTER TABLE mysql.user ADD COLUMN `Show_view_priv` enum('N','Y') CHARACTER SET utf8 NOT NULL DEFAULT 'N' AFTER `Create_view_priv`",
+		"ALTER TABLE mysql.user ADD COLUMN `Create_routine_priv` enum('N','Y') CHARACTER SET utf8 NOT NULL DEFAULT 'N' AFTER `Show_view_priv`",
+		"ALTER TABLE mysql.user ADD COLUMN `Alter_routine_priv` enum('N','Y') CHARACTER SET utf8 NOT NULL DEFAULT 'N' AFTER `Create_routine_priv`",
+		"ALTER TABLE mysql.user ADD COLUMN `Event_priv` enum('N','Y') CHARACTER SET utf8 NOT NULL DEFAULT 'N' AFTER `Create_user_priv`",
+	}
+	for _, sql := range sqls {
+		_, err := s.Execute(sql)
+		if err != nil {
+			if terror.ErrorEqual(err, infoschema.ErrColumnExists) {
+				continue
```
Comment (zimulala) [inline bootstrap.go]: Can we break here?

File: bootstrap.go (hunk 1 above)
Comment (tiancaiamao) [inline bootstrap.go]: If you open it locally, you'll find them all aligned by three tabs.
Seems github can't show the ident correctly.
Should I change those tabs to whitespaces?

File: bootstrap.go (hunk 2 above)
Comment (tiancaiamao) [inline bootstrap.go]: No. we should execute all those sqls above.

**Comment by zimulala** [comment]
LGTM

**Pull request merged**